ansible/playbooks/ptf_installation.yaml

```diff
@@ -5,21 +5,48 @@
   become: true
   become_user: root
   vars:
-    ptf_url: "your_ptf_url"
-    ptf_user: "your_username_here"
-    ptf_password: "your_pass_here"
+    url_timeout: 30
```

Comment:
why we like them to be parametric?
why hardcoding value directly in `Download PTF files with SAS token` is not enough?